subprojects/language-native/src/test/groovy/org/gradle/language/cpp/plugins/CppLibraryPluginTest.groovy

```diff
@@ -28,15 +28,15 @@ import org.gradle.nativeplatform.Linkage
 import org.gradle.nativeplatform.tasks.CreateStaticLibrary
 import org.gradle.nativeplatform.tasks.LinkSharedLibrary
 import org.gradle.test.fixtures.file.TestNameTestDirectoryProvider
-import org.gradle.testfixtures.ProjectBuilder
+import org.gradle.util.TestUtil
 import org.junit.Rule
 import spock.lang.Specification
 
 class CppLibraryPluginTest extends Specification {
```

Comment:
```suggestion
class CppLibraryPluginTest extends AbstractProjectBuilderSpec {
```

All projects using `ProjectBuilder` should extend `AbstractProjectBuilderSpec`. So this test should as well.

I suppose that after this change you won't need to change `TestUtil`.

Reply:
Not only this one, but also:

- `DefaultCppApplicationTest`
- `DefaultCppBinaryTest`
- `DefaultCppComponentTest`
- `DefaultCppLibraryTest`
- `CppApplicationPluginTest`
- ...

And many callers of `TestUtil.createRootProject`. Do you suggest fixing them all?